docs/ebmeds-cloud.md

```diff
@@ -0,0 +1,86 @@
+# EBMEDS 2.0 Cloud
+
+## Endpoints
+
+### `POST /api/v1/cds` - Clinical Decision Support
+
+#### Request
+
+##### Headers
+
+| Header          | Value                                                  |
+| --------------- | ------------------------------------------------------ |
+| `Accept`        | `application/json` OR `application/xml` OR `text/html` |
+| `Authorization` | The user HTTP [bearer token](##authentication).        |
+| `Content-Type`  | `application/json` OR `application/xml`                |
+
+##### Body
+
+The request payload in the format determined by the `Content-Type` header. The payload format must follow the approppriate EBMEDS request JSON or XML schema.
+
+#### Response
+
+The response is one of the following based on the `Accept` header or the _feedback type_ specified in the request:
+
+- Structural JSON response following the EBMEDS response JSON schema.
+- Structural XML response following the EBMEDS response XML schema.
+- Visual HTML response for displaying the structural data directly.
+
+### `POST /api/v1/cmr-app` - Comprehensive Medication Review
+
+#### Request
+
+##### Headers
+
+| Header          | Value                                           |
+| --------------- | ----------------------------------------------- |
+| `Authorization` | The user HTTP [bearer token](##authentication). |
+| `Content-Type`  | `application/x-www-form-urlencoded`             |
+
+##### Body
+
+| Key    | Value                                                   |
+| ------ | ------------------------------------------------------- |
+| `type` | `json` OR `xml`                                         |
+| `data` | The request payload in the format determined by `type`. |
+
+The payload format of `data` must follow the approppriate EBMEDS request JSON or XML schema.
+
+#### Response
+
+Visual HTML response containing the Comprehensive Medication Review for the request payload.
```

Comment:
I like that the structured CMR response is not mentioned.

Reply:
Yeah, I'd like to omit the `/api/v1/cmr` and `/api/v1/dso` endpoints for now, since it isn't really all that well defined. E.g. what's the response schema. 😄